src/DisplayTotalMasteryPage/DisplayTotalMasteryPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Container, makeStyles, Theme, Typography } from '@material-ui/core'
+import React, { FC, useContext, useState } from 'react'
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { GlobalContext } from '../GlobalContext'
+import { theme } from '../theme'
+
+interface DisplayTotalMasteryPageProps {}
+
+const useStyles = makeStyles({
+	resultRow: (theme: Theme) => ({
+		alignSelf: 'stretch',
+		border: '1px solid',
+		borderCollapse: 'collapse',
+		borderColor: theme.palette.primary.main,
+		display: 'flex',
+		justifyContent: 'space-between',
+		marginBottom: theme.spacing(3),
+	}),
+})
+
+const DisplayTotalMasteryPage: FC<DisplayTotalMasteryPageProps> = ({}) => {
+	const { name } = useParams<{ name: string }>()
+	const { api, setTitle } = useContext(GlobalContext) // for page title and API access
+	const classes = useStyles(theme)
+
+	const [result, setResult] = useState({
+		icon: '',
+		mastery: 0,
+		name: '',
+		summonerId: '',
+	})
+
+	const fireGetMastery = async (): Promise<void> => {
+		const summonerResult = await api.pingUserSearchEndpoint(name)
+		const masteryResult = await api.pingMasteryTotalEndpoint(
+			summonerResult.id
+		)
```

Comment:
These two calls can be replaced by a single call w/ the latest API version; more info in the docs - https://simple-nest-staging.herokuapp.com/#/total/MasteryController_getMasteryTotal